tests/test_filters.py

```diff
@@ -665,6 +686,123 @@ def test_glyph_cell_point_data(sphere):
         sphere.glyph(orient='vectors_points', scale='arr_cell', progress_bar=True)
 
 
+class InterrogateVtkGlyph3D:
```

Comment:
This is really nice.  I'm a little hesitant that it will set an expectation for all tests of filters, but maybe it should only be warranted when the behavior is complex like here.

You are a few methods away from creating a class that could be exposed in the public API for a reusable `GlyphFilter`.  I'm not sure that is a good idea however.

Reply:
I don't think there is a precedent for a public PyVista class which wraps a filter - except in the readers. I am fine to leave this in test world for now. The `.glyph` filter has unusually complex behavior which makes this worth doing here.

Reply:
Drive-by bikeshed: we have things like `VTKVersionError` so I'd use `VTK` in this name too.